frontend/app/player/MessageDistributor/network/loadFiles.ts

```diff
@@ -41,28 +39,32 @@ export const loadFiles = (
   })
 }
 
-export const checkUnprocessedMobs = async (sessionId: string) => {
-  try {
-    const api = new APIClient()
-    const res = await api.fetch(getUnprocessedFileLink(sessionId))
-    if (res.status >= 400) {
-      throw NO_UNPROCESSED_FILES
-    }
-    const byteArray = await processAPIStreamResponse(res, false)
-    return byteArray
-  } catch (e) {
-    throw e
+
+export async function requestEFSDom(sessionId: string) {
+  return await requestEFSMobFile(sessionId, "dom.mob")
+}
+
+export async function requestEFSDevtools(sessionId: string) {
+  return await requestEFSMobFile(sessionId, "devtools.mob")
+}
+
+async function requestEFSMobFile(sessionId: string, filename: string) {
+  const api = new APIClient()
+  const res = await api.fetch('/unprocessed/' + sessionId + '/' + filename)
```

Comment:
Ne ochevidno, add Unprocessed to func name maybe? Should be easier to get from the first glance

Reply:
@sylenien Just thought same about "unprocessed" - wtf it is supposed to mean?

Reply:
means that we're grabbing unprocessed/unsaved file from session that's still in progress. Idk, something is better than nothing